app/scripts/filters/canI.js

```diff
@@ -29,6 +29,9 @@ angular
       'imageStreams': [
         {group: '', resource: 'imagestreams', verbs: ['update', 'delete']}
       ],
+      'instances': [
+        {group: 'servicecatalog.k8s.io', resource: 'instances', verbs: ['update', 'create', 'delete']}
```

Comment:
```js
{group: 'servicecatalog.k8s.io', resource: 'instances', verbs: ['update', 'delete']}
```

Eventually we will need to check bindings too if we let users create bindings from the actions menu.

Reply:
Remove `'create'` from the array here. You can create instances on this page.